project/plugins.sbt

```diff
@@ -8,5 +8,5 @@ addSbtPlugin("com.github.cb372"                  % "sbt-explicit-dependencies" %
 addSbtPlugin("de.heikoseeberger"                 % "sbt-header"                % "5.2.0")
 addSbtPlugin("ch.epfl.scala"                     % "sbt-bloop"                 % "1.3.4")
 addSbtPlugin("com.dwijnand"                      % "sbt-dynver"                % "4.0.0")
-addSbtPlugin("com.jsuereth"                      % "sbt-pgp"                   % "1.1.2")
+addSbtPlugin("com.jsuereth"                      % "sbt-pgp"                   % "2.0.0")
```

Comment:
As I mentioned in the other project, we need to stay with sbt-pgp 1.1.2 because version 2.0.0 doesn't work with our current CI setup.